src/main/kotlin/com/vk/admstorm/diagnostic/SentryErrorReporter.kt

```diff
@@ -0,0 +1,65 @@
+package com.vk.admstorm.diagnostic
+
+import com.intellij.diagnostic.IdeaReportingEvent
+import com.intellij.ide.DataManager
+import com.intellij.openapi.actionSystem.CommonDataKeys
+import com.intellij.openapi.diagnostic.ErrorReportSubmitter
+import com.intellij.openapi.diagnostic.IdeaLoggingEvent
+import com.intellij.openapi.diagnostic.SubmittedReportInfo
+import com.intellij.openapi.diagnostic.SubmittedReportInfo.SubmissionStatus
+import com.intellij.openapi.project.Project
+import com.intellij.util.Consumer
+import com.vk.admstorm.notifications.AdmNotification
+import com.vk.admstorm.services.SentryService
+import com.vk.admstorm.utils.MyUtils
+import io.sentry.protocol.SentryId
+import java.awt.Component
+
+class SentryErrorReporter : ErrorReportSubmitter() {
+    override fun getReportActionText(): String {
+        return "Report to Sentry"
+    }
+
+    override fun submit(
+        events: Array<out IdeaLoggingEvent>,
+        additionalInfo: String?, parentComponent: Component,
+        consumer: Consumer<in SubmittedReportInfo>
+    ): Boolean {
+        val project = CommonDataKeys.PROJECT.getData(DataManager.getInstance().getDataContext(parentComponent))
+        if (project == null) {
+            consumer.consume(SubmittedReportInfo(SubmissionStatus.FAILED))
+            return false
```

Comment:
Maybe add here logging like `LOG.warn("cannot get project to submit exception to sentry"")`?

Reply:
We will use `LOG.info`